optimum/commands/export/neuronx.py

```diff
@@ -112,6 +112,12 @@ def parse_args_neuronx(parser: "ArgumentParser"):
         choices=["bf16", "fp16", "tf32"],
         help='The data type to cast FP32 operations to when auto-cast mode is enabled. Can be `"bf16"`, `"fp16"` or `"tf32"`.',
     )
+    optional_group.add_argument(
+        "--tensor_parallel_size",
```

Comment:
They tend to use `tp_degree` in the AWS Neuron SDK documentation and APIs. I used `num_cores` in the decoder, but now I regret it: can't we just align on their terminology ?

Reply:
all three terms were used before the PR; I tried to use the same term as the API I reuse. Indeed, we should agree on one to avoid confusion.
cc. @michaelbenayoun

Reply:
Ok let's adopt tensor_parallel_size for the option name.